server/server.go

```diff
@@ -793,7 +793,9 @@ func (s *Server) startProfile(cpuprofile, memprofile string) {
 		}
 		s.Logger.Printf("I! writing CPU profile to: %s\n", cpuprofile)
 		prof.cpu = f
-		pprof.StartCPUProfile(prof.cpu)
+		if err := pprof.StartCPUProfile(prof.cpu); err != nil {
+			s.Logger.Fatalf("#! start cpu profile: %v", err)
```

Comment:
Hmm, so I see that logger.Fatal is being use here but I'd rather remove it than add to it at this point.  Maybe we leave this little bit as is for now, or setup a clean shutdown mechanism here.